features/financial_assistance/step_definitions/applicant_level_navigation_steps.rb

```diff
@@ -78,4 +78,10 @@
      age_of_applicant >= 19 && current_applicant.is_applying_coverage
     expect(page).to have_content(IvlIapOtherQuestions.primary_caretaker_question_text)
   end
+end
+
+Then(/^the user will navigate to the FAA Family Information page$/) do
```

Comment:
I don't love relying on the copy / text to infer the page location. I would add a specific data-cuke attached to an element instead.